bigquery/api/Snippets/Test.cs

```diff
@@ -37,13 +38,23 @@ public class BigQueryTest : IDisposable, IClassFixture<RandomBucketFixture>
     public BigQueryTest(RandomBucketFixture randomBucketFixture)
     {
         _projectId = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+        _jsonPath = System.Environment.GetEnvironmentVariable("GOOGLE_APPLICATION_CREDENTIALS");
         _client = BigQueryClient.Create(_projectId);
         _storage = StorageClient.Create();
         _bucketName = randomBucketFixture.BucketName;
         _stringOut = new StringWriter();
         Console.SetOut(_stringOut);
     }
 
+    [Fact]
+    public void TestAuthenticateServiceAccount()
+    {
+        var snippet = new BigQueryAuthenticateServiceAccount();
+        snippet.AuthenticateWithServiceAccount(_projectId, _jsonPath);
+        var outputLines = _stringOut.ToString().Trim().Split(new[] { '\n' });
```

Comment:
The params keyword on the `Split (params char[] separator)` declaration means that you don't need to create the char array, you can simply do `.Split('\n')` or even (if you wanted to split on several characters) `.Split(',', ';', ':')`. 
Also, isntead of using '\n' consider using [`Environment.NewLine`](https://docs.microsoft.com/en-us/dotnet/api/system.environment.newline?view=netframework-4.8) so as to have into account the possibility of different new line characters. So it should look like this:
```suggestion
        var outputLines = _stringOut.ToString().Trim().Split(Environment.NewLine);
```

Reply:
Done